dqn.py

```diff
@@ -202,28 +234,40 @@ def main(config: RLConfig) -> None:
     batch_size = config.batch_size
     global_steps = 0
     average_steps = []
+    rewards = []
+    episode_reward = 0
 
     for episode in range(config.num_episodes):
         state = env.reset()
 
         steps = 0
         while True:
-            agent.set_epsilon(linear_epsilon_anneal(global_steps))
+            agent.set_epsilon(
+                linear_epsilon_anneal(global_steps, config.exploration_steps))
+
             action = agent.sample_action(state)
             next_state, reward, done, _ = env.step(action)
             agent.memorize(state, action, reward, next_state, done)
+            episode_reward += reward * (config.discount_factor**steps)
             state = next_state
             steps += 1
 
             if done:
                 average_steps.append(steps)
-                if episode % 20 == 0:
+                rewards.append(episode_reward)
+
+                if (episode + 1) % 20 == 0:
                 # if episode % 1 == 0:
```

Comment:
Maybe also delete the unnecessary commeted code.

Reply:
Sure can do, though I still sometimes reduce the printing to every episode, so that I can debug faster.
Maybe I'll make it a param.